docs/core/logger.md

```diff
@@ -389,6 +390,18 @@ The error will be logged with default key name `error`, but you can also pass yo
 !!! tip "Logging errors and log level"
     You can also log errors using the `warn`, `info`, and `debug` methods. Be aware of the log level though, you might miss those  errors when analyzing the log later depending on the log level configuration.
 
+### Environment variables
```

Comment:
We have this table in the main page ([here](https://docs.powertools.aws.dev/lambda/typescript/latest/#environment-variables)) where we keep track of all env variables, including the ones below. 

Should we just link that one? I'm not sure, what do you think?

Reply:
Yes, let's keep it one place.